R-packages/covidcast/tests/testthat/test-covidcast.R

```diff
@@ -212,7 +214,7 @@ test_that("covidcast_days batches calls to covidcast", {
         stderr = NA,
         sample_size = NA
       )),
-    10)
+    2)
```

Comment:
Why was this previously 10?

Reply:
Mistake when making the test. I was originally was going to simulate 10 days of data, but it seemed like overkill and I changed it to 2 and failed to update the number. It had no impact on the test because the mock only references the first two items of the list, but decided to clean it up since I noticed it now.